### PR TITLE
chore: updates for new deployment schema

### PR DESCRIPTION
**Pull request by menduz**
## What does this PR change?

Buckle up for new release schema

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
           message: |
             After the CI passes:
 
-            - This branch can be previewed at [https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:${{ github.head_ref }}](https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:${{ github.head_ref }}&ENV=org)
+            - This branch can be previewed at https://play.decentraland.zone/?renderer-branch=${{ github.head_ref }}&ENV=org
```
Comment (BrianAmadori) [inline .github/workflows/main.yml:18]: This won't break compatibility with PRs that are created by this week?

**Review by BrianAmadori — APPROVED** [review]

**Review by kuruk-mm — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (menduz) [reply .github/workflows/main.yml:18]: play.zone already points to the new build. More info here: https://github.com/decentraland/explorer-technical-backlog/issues/23

**Comment by kuruk-mm** [comment]
We need to change the README.md to the new format like:

https://play.decentraland.zone/?renderer-branch={BRANCH}&ENV=org

File: README.md [hunk 2]
```diff
@@ -54,13 +54,13 @@ Then, on the Unity editor, click on `Assets > Reimport All`
...
 
-    https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:<branch-name>
+    https://play.decentraland.zone/?renderer-branch=<branch-name>
 
 Note that using this approach, the Unity builds will run against kernel `master` HEAD.
 
 If you want to test your Unity branch against a specific kernel branch, you'll have to use the `renderer` url param like this:
 
-    https://play.decentraland.zone/branch/<kernel-branch-name>/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:<unity-branch-name>
+    https://play.decentraland.zone/branch/<kernel-branch-name>/?renderer-branch=<branch-name>
```
Comment (menduz) [inline README.md:63]:
```suggestion
    https://play.decentraland.zone/?renderer-branch=<branch-name>&kernel-branch=<kernel-branch-name>
```

**Pull request merged**